Tests/NIOTests/NonBlockingFileIOTest.swift

```diff
@@ -915,4 +915,22 @@ class NonBlockingFileIOTest: XCTestCase {
             XCTAssertEqual(numberOfChunks, numberOfCalls.load())
         })
     }
+    
+    func testThrowsErrorOnUnstartedPool() {
+        
+        let fileURL = URL(fileURLWithPath: "/tmp/nonblockingfileiotest.txt")
+        XCTAssertNoThrow(try "hello, world".write(toFile: fileURL.path, atomically: true, encoding: .utf8))
+    
+        let expectation = XCTestExpectation(description: "Opened file")
+        let threadPool = NIOThreadPool(numberOfThreads: 1)
+        let fileIO = NonBlockingFileIO(threadPool: threadPool)
+        let eventLoop = MultiThreadedEventLoopGroup(numberOfThreads: 1).next()
+        fileIO.openFile(path: fileURL.path, eventLoop: eventLoop).whenFailure { (error) in
+            XCTAssertTrue(error is NIOThreadPoolError.ThreadPoolInactive)
+            expectation.fulfill()
+        }
+        
+        eventLoop.shutdownGracefully {_ in }
```

Comment:
We should keep the event loop group around and use `syncShutdownGracefully` on it.